scalatest/src/main/scala/org/scalatest/Engine.scala

```diff
@@ -868,7 +868,7 @@ private[scalatest] class PathEngine(concurrentBundleModMessageFun: => String, si
       else
         count += 1
     }
-    next
+    next.nn
```

Comment:
**Nullable variable is used**
Nullable variable used after `!= null` check. Requires `.nn` to fix.